openbrews/recipe/RecipeUtils.js

```diff
@@ -0,0 +1,131 @@
+angular.module('openbrews.recipeUtils', [])
+.service('RecipeUtils', function () {
+
+    // calculates IBU given a recipe and OG (original gravity).
+    this.calculateIbu = function (recipe, og) {
+        var hops = recipe.hops.filter(function (hop) {
+            return hop.aa;
+        });
+
+        var gravity = this.boilGravity(recipe, og);
```

Comment:
I know this is in the code I linked, but I'm not sure what the difference between boil gravity and og is. Do you know by any chance?